packages/sdk/src/TokenboundClient.ts

```diff
@@ -1,140 +1,89 @@
+} from 'viem'
+import {
+  erc6551AccountAbi,
+  erc6551RegistryAbi,
+  erc1155Abi,
+  erc721Abi,
+  erc20Abi,
+} from '../abis'
+import {
   getAccount,
   computeAccount,
   createAccount,
   getCreationCode,
   prepareExecuteCall,
   executeCall,
-  prepareCreateAccount
+  prepareCreateAccount,
 } from './functions'
-import { 
+import {
   AbstractEthersSigner,
   AbstractEthersTransactionResponse,
-  SegmentedERC6551Bytecode
-} from "./types"
-import { chainIdToChain, segmentBytecode } from "./utils"
-import { normalize } from "viem/ens"
-
-export const NFTTokenType = {
-  ERC721: "ERC721",
-  ERC1155: "ERC1155",
-} as const
-type TokenType = typeof NFTTokenType[keyof typeof NFTTokenType]
-
-type NFTParams = {
-  tokenContract: `0x${string}`
-  tokenId: string
-}
-
-export type TokenboundAccountNFT = NFTParams & {
-  chainId: number
-}
-
-interface TokenTypeParams {
-  tokenType: TokenType
-}
-
-export type NFTTransferParams = TokenTypeParams & NFTParams & {
-  recipientAddress: `0x${string}`
-  account: `0x${string}`
-}
-
-export type ETHTransferParams = {
-  account: `0x${string}`
-  recipientAddress: `0x${string}` // | `${string}.eth`
-  amount: number
-}
-
-export type ERC20TransferParams = {
-  account: `0x${string}`
-  recipientAddress: `0x${string}`
-  amount: number
-  erc20tokenAddress: `0x${string}`
-  erc20tokenDecimals: number
-}
-
-export type TokenboundClientOptions = {
-  chainId: number
-  signer?: any
-  walletClient?: WalletClient
-  publicClient?: PublicClient
-  implementationAddress?: `0x${string}`
-  registryAddress?: `0x${string}`
-}
-
-type Custom6551Implementation = {
-  implementationAddress: `0x${string}`
-  registryAddress?: `0x${string}`
-}
-
-export type TBAccountParams = NFTParams
-
-export type GetAccountParams = TBAccountParams & Partial<Custom6551Implementation>
-export type PrepareCreateAccountParams = TBAccountParams & Partial<Custom6551Implementation>
-export type CreateAccountParams = TBAccountParams & Partial<Custom6551Implementation>
-
-export type ExecuteCallParams = {
-  account: `0x${string}`
-  to: `0x${string}`
-  value: bigint
-  data: string
-}
-
-export type PrepareExecuteCallParams = ExecuteCallParams
-
-export type ComputeAccountParams = TBAccountParams & {
-  chainId: number
-} & Partial<Custom6551Implementation>
-
-export type GetCreationCodeParams = {
-  implementation_: `0x${string}`
-  chainId_: number
-  tokenContract_: string
-  tokenId_: string
-  salt_: string
-}
-
-export type BytecodeParams = {
-  accountAddress: `0x${string}`
-}
+  BytecodeParams,
+  CreateAccountParams,
+  ERC20TransferParams,
+  SignMessageParams,
+  ETHTransferParams,
+  ExecuteCallParams,
+  GetAccountParams,
+  NFTTokenType,
+  NFTTransferParams,
+  PrepareCreateAccountParams,
+  PrepareExecuteCallParams,
+  SegmentedERC6551Bytecode,
+  TokenboundAccountNFT,
+  TokenboundClientOptions,
+} from './types'
+import { chainIdToChain, segmentBytecode } from './utils'
+// import { normalize } from 'viem/ens'
```

Comment:
Is there a reason this is commented out?

Reply:
Yep, just need to actually get it working :)